kDrive/SceneDelegate.swift

```diff
@@ -222,6 +229,17 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate, AccountManagerDel
         }
     }
 
+    @discardableResult
+    private func continueToWebActivityIfPossible(_ scene: UIScene, userActivity: NSUserActivity) async -> Bool {
+        guard userActivity.activityType == NSUserActivityTypeBrowsingWeb,
+              let incomingURL = userActivity.webpageURL else {
+            Log.sceneDelegate("the scene continue userActivity - is not NSUserActivityTypeBrowsingWeb", level: .error)
+            return false
+        }
+
+        return await DeeplinkParser().parse(url: incomingURL)
```

Comment:
We can use the injection now. There is also an other occurence in class SceneDelegate.swift:227
```suggestion
        return await deeplinkParser.parse(url: incomingURL)
```

Reply:
I changed all remaining occurrences of this